components/raftstore/src/store/peer.rs

```diff
@@ -2814,7 +2814,15 @@ where
                 last_index
             ));
         }
-        assert!(min_matched >= min_committed);
+        if min_matched < min_committed {
+            warn!(
+                "min_matched < min_committed, raft progress is inaccurate";
+                "region_id" => self.region_id,
+                "peer_id" => self.peer.get_id(),
+                "min_matched" => min_matched,
+                "min_committed" => min_committed,
+            );
```

Comment:
We can set `min_matched` to `min_committed` to reduce the entry size.

Reply:
Oh. I think we can do this in `get_min_progress` because `min_matched >= min_committed` should always true.

Reply:
And the `warn` log can also be printed in `get_min_progress`.

Reply:
Done.

Reply:
This can be unsafe. The actual `min_match` can be greater than `min_committed`, if resetting inaccurate `min_match` (like 0) to `min_committed` then it may requires more logs that even beyond the first index of existing peer to propose commit merge. Hence lead to panic like #12663.